tests/kalatori-api-test-suite/tests/order.test.ts

```diff
@@ -8,7 +8,7 @@ describe('Order Endpoint Blackbox Tests', () => {
     throw new Error('check all environment variables are defined');
   }
   const dotOrderData = {
-    amount: 1,
+    amount: 2,
```

Comment:
add a comment here that indicates that this should be above mortal

Reply:
resolved in [16fe114](https://github.com/Alzymologist/Kalatori-backend/pull/106/commits/16fe114e333990884ec4beabb69f1e5518c74cde)